packages/in_app_purchase/in_app_purchase_android/lib/src/billing_client_wrappers/billing_client_wrapper.dart

```diff
@@ -324,6 +325,21 @@ class BillingClient {
     return result ?? false;
   }
 
+  /// BillingConfig method channel string identifier.
```

Comment:
Nit: Did you want to match the comment style for all lines? i.e. consistently use either /// on this line vs // on the next line

Reply:
/// Is used for dart doc 
// is used for comments. In this case I want callers to see the contends in /// and maintainers to see the comments after //. 

If you want I can move the developer comments inside the function